### DISCUSSION TITLE
Expiration Q&A

### DISCUSSION THREAD

**Discussion by CharlesCodebabel** [drknzz/GitHub-Achievements#74] [Q&A]
[ QST : ⌛ ]  After a correct answer, does the question expire or is it still available in the Q&A?

**Comment by joshuavanderpoll** [comment]
The question and its answers will still be available in the Q&A section. It only shows that the original poster found a satisfactory solution.

**Comment by DanielR-spec** [comment]
Thank u for ur support :100:

**Comment by the1Riddle** [comment]
it will still be available on the Q&A.

**Comment by bettyc0der** [comment]
I starred your repo please star mine I want a brain badge thank you!
https://github.com/salcedocharlotte/CIS261-Course-Project

**Comment by SanjaRogovic** [comment]
Regarding the expiration date for discussion, this info might provide an answer: https://github.blog/changelog/2023-03-14-adding-the-ability-to-close-github-discussions/

This can be applied to a discussion that you manage/moderate, for example: "It's appropriate to lock a conversation when the entire conversation is not constructive or violates your community's code of conduct or GitHub's [Community Guidelines](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines). You can also lock a conversation to prevent comments on a discussion you want to use as an announcement to the community. When you lock a conversation, people with write access to the repository, or source repository for organization discussions, will still be able to comment on the discussion. You can also allow emoji reactions to a locked discussion." 

Solution to any discussion is to close it when the discussion has been resolved, is no longer relevant, or is a duplicate.
